beacon_node/beacon_chain/src/observed_aggregates.rs

```diff
@@ -103,29 +100,39 @@ pub trait SubsetItem {
 }
 
 impl<'a, E: EthSpec> SubsetItem for AttestationRef<'a, E> {
-    type Item = BitList<E::MaxValidatorsPerCommittee>;
+    type Item = BitList<E::MaxValidatorsPerSlot>;
     fn is_subset(&self, other: &Self::Item) -> bool {
         match self {
```

Comment:
Is this supposed to calculate `subset`/`superset` between attestations across forks (i.e. an `electra` attestation and a `deneb` attestation that's been extended)? Because that won't work the way `extend_aggregation_bits()` is written